From 4c202c0fd7e5bd3264c720d715f05573047d9cf5 Mon Sep 17 00:00:00 2001
From: Erico Nunes <nunes.erico@gmail.com>
Date: Fri, 12 Aug 2016 23:11:56 +0200
Subject: [PATCH] fwts: do not use -Werror

Warnings come and go with various compiler versions, so using -Werror is
prone to cause build failures with various compiler versions, especially
newer versions that introduce new warnings.

Remove use of -Werror.

Signed-off-by: Erico Nunes <nunes.erico@gmail.com>
[Fabrice: updated for 19.09.00]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 configure.ac              | 2 +-
 src/Makefile.am           | 2 +-
 src/lib/src/Makefile.am   | 2 +-
 src/utilities/Makefile.am | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/configure.ac b/configure.ac
index 0109d644..3ca599e1 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,5 +1,5 @@
 	  AC_INIT([fwts],[0.1],[fwts-devel@lists.ubuntu.com])
-          AM_INIT_AUTOMAKE([-Wall -Werror foreign subdir-objects])
+          AM_INIT_AUTOMAKE([-Wall foreign subdir-objects])
           m4_ifdef([AM_PROG_AR], [AM_PROG_AR])
 	  AC_CANONICAL_HOST
 	  AC_CONFIG_MACRO_DIR([m4])
diff --git a/src/Makefile.am b/src/Makefile.am
index 52c637dd..d9d73f1c 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -14,7 +14,7 @@ AM_CPPFLAGS = \
 	`pkg-config --silence-errors --cflags json` \
 	`pkg-config --silence-errors --cflags json-c` \
 	-pthread `pkg-config --cflags glib-2.0 gio-2.0` \
-	-Wall -Werror -Wextra				\
+	-Wall -Wextra					\
 	-Wno-address-of-packed-member
 
 bin_PROGRAMS = fwts
diff --git a/src/lib/src/Makefile.am b/src/lib/src/Makefile.am
index 54de7f44..e129ed16 100644
--- a/src/lib/src/Makefile.am
+++ b/src/lib/src/Makefile.am
@@ -24,7 +24,7 @@ AM_CPPFLAGS = \
 	`pkg-config --silence-errors --cflags json-c`	\
 	`pkg-config --cflags glib-2.0 gio-2.0` 		\
 	-DDATAROOTDIR=\"$(datarootdir)\"		\
-	-Wall -Werror -Wextra				\
+	-Wall -Wextra					\
 	-Wno-address-of-packed-member
 
 pkglib_LTLIBRARIES = libfwts.la
diff --git a/src/utilities/Makefile.am b/src/utilities/Makefile.am
index aa37de55..4d5297cc 100644
--- a/src/utilities/Makefile.am
+++ b/src/utilities/Makefile.am
@@ -16,7 +16,7 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 #
 
-AM_CPPFLAGS = -Wall -Werror -Wextra -DDATAROOTDIR=\"$(datarootdir)\" \
+AM_CPPFLAGS = -Wall -Wextra -DDATAROOTDIR=\"$(datarootdir)\" \
 	`pkg-config --silence-errors --cflags json` \
 	`pkg-config --silence-errors --cflags json-c`
 
-- 
2.17.1

